testapi.pm

```diff
@@ -601,7 +601,7 @@ Example:
 =cut
 
 sub assert_screen_change(&@) {
-    ::wait_screen_change(@_) or die 'assert_screen_change failed to detect a screen change';
+    wait_screen_change(\&{shift()}, @_) or die 'assert_screen_change failed to detect a screen change';
```

Comment:
I guess you will still need to declare the method, or declare assert_screen_change after wait_screen_change so that the tests don't fail (but let's see)

Reply:
`wait_screen_change` is declared above `assert_screen_change`, so should be fine.

Reply:
Done. Was to motivated to keep it as single line =)))